docker/demo/config/log4j.properties

```diff
@@ -25,6 +25,8 @@ log4j.appender.console.layout.ConversionPattern=%d{yy/MM/dd HH:mm:ss} %p %c{1}:
 # log level for this class is used to overwrite the root logger's log level, so that
 # the user can have different defaults for the shell and regular Spark apps.
 log4j.logger.org.apache.spark.repl.Main=WARN
+# Adjust Hudi internal logging levels
+log4j.logger.org.apache.hudi=DEBUG
```

Comment:
sure i can remove this, this was more helpful for seeing the logging for docker IT tests and I think will help in general people debug. If the concern is that it will generate too many logs for azure ci IT section then we can remove this.